File: cfgs/java_cfgs_foo/STACK_PERMUTATIONS_CHECK_IF_AN_ARRAY_IS_STACK_PERMUTATION_OF_OTHER.java
Nodes
1;special;;;
2;statement;Queue<Integer> input = new LinkedList<>();14;
3;statement;int i = 0;15;
4;conditional;foo_1 < foo_2;16;
5;statement;fun_1(foo_2[foo_3]);18;
6;statement;i++;17;
7;statement;Queue<Integer> output = new LinkedList<>();20;
8;statement;int i = 0;21;
9;conditional;foo_1 < foo_2;22;
10;statement;fun_1(foo_2[foo_3]);24;
11;statement;i++;23;
12;statement;Stack<Integer> tempStack = new Stack<>();26;
13;conditional;!fun_1();27;
14;statement;int ele = fun_1();28;
15;conditional;foo_1 == fun_1();29;
16;statement;fun_1();30;
17;conditional;!fun_1();31;
18;conditional;fun_1() == fun_2();32;
19;statement;fun_1();33;
20;statement;fun_1();34;
21;statement;break;36;
22;statement;fun_1(foo_2);40;
23;statement;return fun_1() && fun_2();43;
24;exit;;;
Edges
0;2;
1;3;
2;4;
3;5;7;
4;6;
5;4;
6;8;
7;9;
8;10;12;
9;11;
10;9;
11;13;
12;23;14;
13;15;
14;16;22;
15;17;
16;13;18;
17;19;21;
18;20;
19;17;
20;13;
21;13;
22;24;
23;

File: cfgs/pythoncfgs/STACK_PERMUTATIONS_CHECK_IF_AN_ARRAY_IS_STACK_PERMUTATION_OF_OTHER.py
Nodes
1;statement;foo_1 = fun_1();10;
2;conditional;fun_1(foo_1);11;
3;statement;foo_1 = fun_1();13;
4;conditional;fun_1(foo_1);14;
5;statement;foo_1 = [];16;
6;conditional;not fun_1();17;
7;statement;return fun_1() and fun_2(foo_1) == 0;30;
8;statement;foo_1 = foo_2.queue[0];18;
9;statement;fun_1();19;
10;conditional;foo_1 == foo_2.queue[0];20;
11;statement;fun_1(foo_1);29;
12;statement;fun_1();21;
13;conditional;fun_1(foo_1) != 0;22;
14;conditional;foo_1[-1] == foo_2.queue[0];23;
15;statement;fun_1();24;
16;statement;fun_1();25;
17;statement;fun_1(foo_1[foo_2]);15;
18;statement;fun_1(foo_1[foo_2]);12;
19;exit;;
Edges
0;2;
1;3;18;
2;4;
3;5;17;
4;6;
5;7;8;
6;19;
7;9;
8;10;
9;11;12;
10;6;
11;13;
12;6;14;
13;6;15;
14;16;
15;13;
16;4;
17;2;
18;